arboretum/ibm_cloud/README.md

```diff
@@ -18,10 +18,62 @@ how to include the fetchers and checks from this library in your downstream proj
 
 ## Fetchers
 
-Fetchers coming soon...
+### Cluster List
+
+* Class: [ClusterListFetcher][fetch-cluster-list]
+* Purpose: Write the list of IBM Cloud clusters to the evidence locker.
+* Behavior: Log in to IBM Cloud using `ibmcloud login` command, and save the result of `ibmcloud cs cluster ls` command.
+* Expected configuration elements:
+  * org.ibm_cloud.cluster_list.config
+    * List of objects representing the IKS accounts
+    * Each object must have the following values
+      * `account` - a list containing names identifying the IKS account, this will map to an IAM token provided in the credentials file
+* Expected configuration example:
+   ```json
+   {
+     "org": {
+       "ibm_cloud": {
+         "cluster_list": {
+           "config": {
+               "account": ["myaccount1"]
+           }
+         }
+       }
+     }
+   }
+   ```
+* <a name="expected_credentials"></a>Expected credentials:
+  * `ibm_cloud` credentials with read/view permissions are needed for this fetcher to successfully
+   retrieve the evidence.
+    * One IKS API key is required per account specified in the configuration.  See above.  Each account provided in the configuration must preceed `_api_key`.  For example, if we have specified accounts "acct_a", "acct_b", and "acct_c" your configuration should look like:
+
+      ```ini
+      [ibm_cloud]
+      acct_a_api_key=your-iks-api-key-for-acct-a
+      acct_b_api_key=your-iks-api-key-for-acct-b
+      acct_c_api_key=your-iks-api-key-for-acct-c
+      ```
+
+    * Expected Travis environment variable settings to generate credentials:
+      * `IBM_CLOUD_ACCT_A_API_KEY`
+      * `IBM_CLOUD_ACCT_B_API_KEY`
+      * `IBM_CLOUD_ACCT_C_API_KEY`
+
+  * NOTE: [API Keys can be generated using the ibmcloud CLI][ic-api-key-create]. E.g.
```

Comment:
I think we also need to add a note that stipulates that this fetcher expects that the ibmcloud CLI is installed in order to function.  You'll need to provide a link to ibmcloud docs that describe installing the CLI.  Maybe also mention that in order for this to work when running in CI/CD that consumers of this fetcher will have to ensure that the CLI is installed in the CI/CD execution environment.

Reply:
Judging by the code, you should also ensure that the Kubernetes service plugin is also installed.  You should make a note of that in the README as well.

Reply:
I mentioned the prerequisites as notes in the [README](https://github.com/ComplianceAsCode/auditree-arboretum/pull/15/files#diff-fa9c900ac18eaea9441dab042c0ad0c9).
Is there better position to describe such a prerequisites?